### PR TITLE
Add support for CTRL+S to send hardware "shake" to device

### PR DESCRIPTION
**Pull request by atrich**
When developing for React Native, shaking the physical Android device is how you pull up the React Native debug menu, which is a very common task.

This catches the CTRL+S key combination and maps it to the "shake" keycode (keycode 82, AKEYCODE_MENU).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,6 +258,7 @@ To run without installing:
  | click on `HOME`                        | `Ctrl`+`h` \| _Middle-click_  |
  | click on `BACK`                        | `Ctrl`+`b` \| _Right-click²_  |
  | click on `APP_SWITCH`                  | `Ctrl`+`m`                    |
+ | shake device                           | `Ctrl`+`s`                    |
```
Comment (rom1v) [inline README.md]: For consistency:

> click on `MENU`

Comment (rom1v) [reply README.md]: There are also shortcut descriptions for `scrcpy --help` in `main.c`. Could you also add one for `MENU` below `APP_SWITCH` like here?

**Comment by rom1v** [comment]
@atrich Do you want me to apply the requested changes for you?

**Comment by rom1v** [comment]
Thank you :+1:

I also swapped the shortcuts to avoid confusion: https://github.com/Genymobile/scrcpy/commit/88ee6bc928e82a407a976665b1401cf23c3879df

**Comment by ihwf** [comment]
This can not work on MacOS! anybody help me?

**Comment by ihwf** [comment]
> Thank you 👍
> 
> I also swapped the shortcuts to avoid confusion: [88ee6bc](https://github.com/Genymobile/scrcpy/commit/88ee6bc928e82a407a976665b1401cf23c3879df)

This can not work on my MacOS ,  short press ctrl+M trigger APP_SWITCH, the same as command+S,  and  long press ctrl+M well trigger shake device , I don't know why must be long press?

**Comment by rom1v** [comment]
> short press ctrl+M trigger APP_SWITCH, the same as command+S, and long press ctrl+M well trigger shake device

Actually, `Ctrl`+`m` is for `MENU`, and `Cmd`+`s` is for `APP_SWITCH`:
https://github.com/Genymobile/scrcpy/blob/8ec077ce1b5ae3c8a5993e062668220e7c684c30/app/src/main.c#L157-L161
https://github.com/Genymobile/scrcpy/blob/8ec077ce1b5ae3c8a5993e062668220e7c684c30/app/src/input_manager.c#L310-L321

I don't know what "shake device" is, but for example in the VLC app, `Ctrl`+`m` opens the menu and `Ctrl`+`s` (`Cmd`+`s` on macOS) correctly move into the "app switch" mode.

Note that you can inject these events manually via `adb`:
```bash
adb shell input keyevent APP_SWITCH
adb shell input keyevent MENU
```

**Comment by ihwf** [comment]
> > short press ctrl+M trigger APP_SWITCH, the same as command+S, and long press ctrl+M well trigger shake device
> 
> Actually, `Ctrl`+`m` is for `MENU`, and `Cmd`+`s` is for `APP_SWITCH`:
> https://github.com/Genymobile/scrcpy/blob/8ec077ce1b5ae3c8a5993e062668220e7c684c30/app/src/main.c#L157-L161
> 
> 
> https://github.com/Genymobile/scrcpy/blob/8ec077ce1b5ae3c8a5993e062668220e7c684c30/app/src/input_manager.c#L310-L321
> 
> I don't know what "shake device" is, but for example in the VLC app, `Ctrl`+`m` opens the menu and `Ctrl`+`s` (`Cmd`+`s` on macOS) correctly move into the "app switch" mode.
> 
> Note that you can inject these events manually via `adb`:
> 
> ```shell
> adb shell input keyevent APP_SWITCH
> adb shell input keyevent MENU
> ```

I mean  short press `Ctrl `+`m` handle "app switch" on my MacOS, it's wrong, I want to opens the menu, and when I long press `Ctrl `+`m` , it can opens the menu, I don't know why long press is right

**Comment by rom1v** [comment]
> I mean short press Ctrl +m handle "app switch" on my MacOS

I don't know how it's even possible, looking at the source code. :confused: 

What is the result of `scrcpy -v`?

**Comment by ihwf** [comment]
> I don't know how it's even possible, looking at the source code. 😕
> 
> What is the result of `scrcpy -v`?

`scrcpy 1.11

dependencies:
 - SDL 2.0.10
 - libavcodec 58.54.100
 - libavformat 58.29.100
 - libavutil 56.31.100`

It's a strange problem, maybe it's not a general problem, have you test on MacOS ?

**Comment by r4dixx** [comment]
> This can not work on my MacOS , short press ctrl+M trigger APP_SWITCH, the same as command+S, and long press ctrl+M well trigger shake device , I don't know why must be long press?

@ihwf Haven't tested this feature, but `ctrl + m` and `cmd + s` work as expected here. Maybe you have some keyboard conflicts linked to custom keybindings and/or locale settings? Just a supposition.

`scrcpy 1.11`

`dependencies:`
 `- SDL 2.0.10`
` - libavcodec 58.54.100`
` - libavformat 58.29.100`
 `- libavutil 56.31.100`

`Darwin parprpmc008675a 18.7.0 Darwin Kernel Version 18.7.0: Sat Oct 12 00:02:19 PDT 2019; root:xnu-4903.278.12~1/RELEASE_X86_64 x86_64`

`ProductName:    Mac OS X`
`ProductVersion: 10.14.6`
`BuildVersion:   18G1012`

**Pull request merged**